app/controllers/staff/session_formats_controller.rb

```diff
@@ -30,7 +32,11 @@ def create
   end
 
   def update
-    @session_format.update_attributes(session_format_params)
+    if @session_format.update_attributes(session_format_params)
```

Comment:
This is a little bit off track, but I just noticed that we're passing :event_id in through the form instead of using current_session to set the format. I can stick that into a separate card no prob, but if you wanted to fix that in this PR, that works too.